extensions/typescript-language-features/package.nls.json

```diff
@@ -8,6 +8,7 @@
 	"configuration.suggest.completeFunctionCalls": "Complete functions with their parameter signature.",
 	"configuration.suggest.includeAutomaticOptionalChainCompletions": "Enable/disable showing completions on potentially undefined values that insert an optional chain call. Requires strict null checks to be enabled.",
 	"configuration.suggest.includeCompletionsForImportStatements": "Enable/disable auto-import-style completions on partially-typed import statements.",
+	"typescript.experimental.aiQuickFix": "Enable/disable AI-assisted quick fixes.",
```

Comment:
Should we note that this requires copilot?

Reply:
Maybe but w/o mentioning copilot since technically this can be any extension